src/DayTime.cpp

```diff
@@ -294,6 +295,11 @@ const char *DayTime::formatStringImpl(char *targetBuffer, const char *format, ch
                     {
                         switch (macro)
```

Comment:
More of a general question, but why are we doing our own string formatting? Is this stuff specific to an interface?

Reply:
Not sure what you mean? As opposed to using sprintf()?

Reply:
Yeah, the standard library of string formatting functions

Reply:
That was two years ago, I can't remember yesterdays breakfast.... there was a reason in that I couldn't get it to work correctly on the Arduino. I don't remember why, though.

Reply:
Lol fair enough, if it's not interface specific I may look into removing this stuff and changing over to standard string formatting